pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -342,19 +356,6 @@ else let
       outputHashAlgo = attrs.outputHashAlgo or "sha256";
       outputHashMode = attrs.outputHashMode or "recursive";
     } // lib.optionalAttrs (stdenv.hostPlatform != stdenv.buildPlatform) {
-      cmakeFlags =
```

Comment:
Heh, in this diff it looks like it was previously only applied when cross-compiling?

Reply:
Yes